driver-core/src/test/unit/com/mongodb/async/CallbackResultHolder.java

```diff
@@ -35,13 +38,21 @@
      */
     public void onResult(final T result, final Throwable error) {
         if (isDone) {
-            throw new IllegalStateException("The CallbackResult cannot be initialized multiple times.");
+            throw new IllegalStateException("The CallbackResult cannot be initialized multiple times.  The first time it was initialized "
```

Comment:
Did this when debugging retryable writes test failure, but seems generally useful.